src/components/theme/View/ListingView.jsx

```diff
@@ -28,11 +28,13 @@ const ListingView = ({ content }) => (
             {item.description && <p>{item.description}</p>}
           </Container>
           {item.image_field && (
-            <PreviewImage
+            <Component
+              componentName="PreviewImage"
               item={item}
               size="thumb"
-              alt={item.image_caption ? item.image_caption : item.title}
```

Comment:
We recently had an a11y review from a specialized company that, among other things, suggested that we don't add the image title as alt because it's already there, right before or after the image, so an empty alt is better.
On the other hand, as you suggested in other comments, we should keep the `image_caption` as alt when provided, so I am adding it back, but I would not use the title as fallback.

Reply:
@pnicolli Okay, I understand the logic now: don't reuse text for the `alt` that is already displayed nearby. Makes sense.

Reply:
Fixed

Reply:
We don't really have an `alt` thingie here from the block configuration or the item, we just have the optional `image_caption` field that we use here.
I am not sure I fully understand your question. The `alt` will be read together with the image, always, so the caption will always be in the same context. The item title, on the other hand, can be before or after based on the listing variation, but in any case the screen reader will read the elements in the order they are rendered and they will always be listed in the same order in this block, so it should be fairly easy to grasp.